niceman/distributions/docker.py

```diff
@@ -123,11 +123,10 @@ def identify_distributions(self, files):
                 image = json.loads(self._session.execute_command(['docker',
                     'image', 'inspect', file])[0])[0]
 
-                # Fail if the Docker image has not been pushed to a repository.
-                # If the image is not in a repository, then it can't be
-                # expected to be reasonably discovered by others to reproduce.
+                # Warn user if the image does not have any RepoDigest entries.
                 if not image['RepoDigests']:
-                    raise CommandError(msg='No Docker repos found')
+                    lgr.warning("The Docker image '{}' does not have any \
+repository IDs associated with it".format(file))
```

Comment:
Very minor, but just a note.  With logging messages, it's preferable to use something like `lgr.warning("a %s", foo)` because that avoids unnecessarily formatting the string when the logger isn't enabled for that level.

Reply:
Thanks for the tip! I've made the change.